PDO/PDO.php

```diff
@@ -710,6 +710,18 @@ class PDO  {
 	 */
 	const MYSQL_ATTR_MULTI_STATEMENTS = 1015;
 
+	/**
+	 * <p>
+	 * Disables SSL peer verification when set to FALSE.
+	 * </p>
+	 * <p>
+	 * Note, this constant is not yet documented on the php.net documentation.
+	 * </p>
+	 * @since 7.0.18 and 7.1.4
```

Comment:
This should be splitted to two `@since` tags

Reply:
I can split them but what about this then ?
https://github.com/JetBrains/phpstorm-stubs/blob/master/PDO/PDO.php#L708

Reply:
dunno 🤷‍♂️  If this ever become parsed, then it's better to have them splitted

Reply:
Also, take a look at `@since` definition at http://docs.phpdoc.org/references/phpdoc/tags/since.html

Reply:
Okay, I'll fix mine + another PR to fix others, @andrey-sokolov what's your mind on this ?

Reply:
For new contributions it's better ti split such tags